genai-perf/genai_perf/config/input/config_command.py

```diff
@@ -207,10 +207,6 @@ def _check_payload_input(self) -> None:
                         f"User Config: perf_analyzer.stimulus: {key} is not supported with the payload input source."
                     )
 
-        if self.perf_analyzer.get_field("warmup_request_count").is_set_by_user:
-            raise ValueError(
-                f"User Config: perf_analyzer.warmup_request_count is not supported with the payload input source."
-            )
```

Comment:
As mentioned earlier, GAP used to not support using fixed schedule mode with the warmup feature, hence this error message.

Now, it does, so the error message/check is removed.